### ISSUE TITLE
Allow importing DB layers into a Kart repo

### ISSUE THREAD

**Issue by volaya** [koordinates/kart-qgis-plugin#56]
Currently, when importing a layer, the plugin shows a file selector dialog. It should give the option to import from a file or from a database connection, so users can import any data they might have, without needing to manually convert it to a file

**Comment by jguelat** [comment]
One interesting solution would be to use the QGIS browser as a file selector dialog. This would allow selecting files, tables in databases, tables in GeoPackages, etc.

**Comment by jguelat** [comment]
No, I mean the standard browser (View -> Panels -> Browser). I don't know if it's possible in a QGIS Python plugin, but it would be amazing to have a "separate" browser window to select the files/database tables that need to be imported into the kart repositories.
